share/service/service.go

```diff
@@ -0,0 +1,142 @@
+package service
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/celestiaorg/celestia-node/share"
+
+	"github.com/celestiaorg/celestia-node/share/retriever"
+
+	"golang.org/x/sync/errgroup"
+
+	"github.com/ipfs/go-blockservice"
+	"github.com/ipfs/go-cid"
+
+	"github.com/celestiaorg/celestia-node/share/ipld"
+	"github.com/celestiaorg/nmt"
+	"github.com/celestiaorg/nmt/namespace"
+)
+
+// Share is a fixed-size data chunk associated with a namespace ID, whose data will be erasure-coded and committed
+// to in Namespace Merkle trees.
+type Share = share.Share
```

Comment:
This alias is now unnecessary and can be removed in favour of `share.Share`

Reply:
Ty, removed it.